linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -805,32 +795,22 @@ private[emitter] final class ClassEmitter(sjsGen: SJSGen) {
             genIsInstanceOfHijackedClass(obj, className)
 
           case _ =>
-            var test = if (tree.kind.isClass) {
-              genIsInstanceOfClass(obj, className)
-            } else {
-              !(!(
-                  genIsScalaJSObject(obj) &&
-                  genIsClassNameInAncestors(className,
-                      obj DOT "$classData" DOT "ancestors")
-              ))
+            val baseTest = {
+              if (tree.kind.isClass) {
```

Comment:
There's a spurious diff of several lines here, which could be avoided by keeping
```suggestion
            val baseTest = if (tree.kind.isClass) {
```